README.md

```diff
@@ -199,6 +199,55 @@ An error will be raised:
 
 `ActiveModel::ValidationError: Validation failed: Title can't be blank`
 
+#### Content Areas
+
+A component can be configured to have multiple 'content areas' which can be captured and later rendered in the component's view. For example:
+
+`app/components/modal_component.rb`:
+```ruby
+class ModalComponent < ActionView::Component::Base
+  validates :user, :header, :body, presence: true
+
+  with_content_areas :header
+
+  def initialize(user:)
+    @user = user
+  end
+end
+```
+
+`app/components/modal_component.html.erb`:
+```erb
+<div class="modal">
+  <div class="header"><%= header %></div>
+  <div class="body"><%= body %>"></div>
+</div>
+```
+
+We can render it in a view as:
+
+```erb
+<%= render(TestComponent, user: {name: 'Jane'}) do |component| %>
+  <% component.with(:header) do %>
+      Hello <%= user[:name] %>
```

Comment:
Wouldn't this need to be `@user`?

```suggestion
      Hello <%= @user[:name] %>
```

Reply:
No this is valid. All the methods on the components are in scope